packages/nc-gui-v2/pages/index/index/create-external.vue

```diff
@@ -77,21 +90,43 @@ const validators = computed(() => {
 
 const { validate, validateInfos } = useForm(formState, validators)
 
+const populateName = (v: string) => {
+  formState.dataSource.connection.database = `${v?.trim()}_noco`
+}
+
 const onClientChange = () => {
   formState.dataSource = { ...getDefaultConnectionConfig(formState.dataSource.client) }
+  populateName(formState.title)
 }
 
-const inflectionTypes = ['camelize', 'none']
-const configEditDlg = ref(false)
+const onSSLModeChange = (v: any) => {
```

Comment:
Should not be `any`. 
Either use an enum or just use `v: 'No' | 'Allowed' | string`
If possible don't use capitalized values either and change it to `no` and `allowed` or (just use a boolean value instead?)

Reply:
This causes @select event to fire a type error, what should I use to avoid this
![image](https://user-images.githubusercontent.com/59797957/186239585-d0de227d-1bb1-4b46-9dc3-96ce1dd77353.png)

Reply:
I opened a separate PR with proposed type fixes #3307 
If it's okay with you, you can merge that into this PR 😄